ska_builder.py

```diff
@@ -40,14 +40,7 @@
 BUILD_LIST = [b for b in BUILD_LIST if not re.match(r"^\s*$", b)]
 
 if platform.uname().system == "Darwin":
-    os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.9"
-
-if platform.uname().machine == 'i686':
-    # Skip starcheck and ska3-perl on 32 bit
-    for pkg in ['starcheck', 'ska3-perl']:
-        if pkg in BUILD_LIST:
-            BUILD_LIST.remove(pkg)
-
+    os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.14"  # Mojave
```

Comment:
I think this wasn't really working in the old version of conda anyway (not really setting to 10.9).  If the variable works now, will probably need to scrub the twiki docs for all references of machine `zero` as I don't think the imac supports mojave.

Reply:
Granted, if @javierggt 's OSX building is ready-to-go all the `zero` stuff is probably moot anyway.  Just don't have good ways to capture changes to docs in the accompanying code with use of the wiki.

Reply:
Why was this in there to begin with?  Can we just cut it out?

Reply:
The plan with build automation is that the wiki gets chopped down in a big way.  But if you wanted to, it is easy enough to branch the current maintainers wiki into a shiny version and start editing that to reflect how things will be post-shiny.